binance/client.py

```diff
@@ -506,7 +506,28 @@ def get_symbol_info(self, symbol) -> Optional[Dict]:
                 return item
 
         return None
+    
+    def get_latest_price(self,symbol) -> int:
+        """Latest price for One specific symbols.
 
+        Reference Document : https://binance-docs.github.io/apidocs/spot/en/#symbol-price-ticker
+
+        :returns: Latest Price of a coin
+        
+        .. code-block:: python
+
+            input : BNBUSDT
+            output : 600.2 
+
+        :raises: BinanceRequestException, BinanceAPIException
+
+        """
+        val=0.0
+        list_price= [x['price'] for x in self.get_all_tickers() if x['symbol']==symbol]
+        
+        for i in list_price:
+            val=float(i)
```

Comment:
Can we simplify this, a for loop doesn't seem necessary if we are going to have one result.

It should also raise an exception if the symbol isn't found not return 0.

Reply:
Yes, For loop is overkill. What I was aiming for was for mathematical calculations, the exact price of a symbol at an instant.  I am removing the For loop but can I keep the return type as "float", or you want me to make it a string and yes, if symbol not found, it will return 0, making that an exception.